docs/api/permissions.rst

```diff
@@ -0,0 +1,332 @@
+.. _api-permissions:
+
+Permissions
+###########
+
+As described in the :ref:`concepts section <concepts-permissions>`, permissions
+can be set on any object.
+
+Formalism
+=========
+
+On each kind of object the set of permissions can be:
+
++------------+-----------------------------------------+
+| Permission | Description                             |
++============+=========================================+
+| **read**   | Any listed :term:`principal` can read   |
+|            | the object.                             |
++------------+-----------------------------------------+
+| **write**  | Any listed :term:`principal` can write  |
+|            | the object. Whoever has the permission  |
+|            | to write an object can read, update and |
+|            | delete it.                              |
++------------+-----------------------------------------+
+| **create** | Any listed :term:`principal` can create |
+|            | a new *child object*.                   |
++------------+-----------------------------------------+
+
+In the case of a creation, since an object can have several kinds of children, the
+permission is prefixed (for instance ``groups:create``, ``collections:create``).
+
+The following table lists all permissions that can be associated to each kind
+of object.
+
++----------------+------------------------+----------------------------------+
+| Object         | Associated permissions | Description                      |
++================+========================+==================================+
+| Configuration  | ``buckets:create``     | Ability to create new buckets.   |
```

Comment:
The construct "Ability to" doesn't need to be used (or repeated) in this list and should be removed. Consider:
> Create new buckets.
Read all objects in the bucket.
Create new records in the collection.
etc.